ui/component/livestreamComment/view.jsx

```diff
@@ -59,6 +68,11 @@ export default function LivestreamComment(props: Props) {
 
   const [hasUserMention, setUserMention] = React.useState(false);
 
+  const shouldFetchUserMemberships = true;
+  useGetUserMemberships(shouldFetchUserMemberships, [authorUri], claimsByUri, doFetchUserMemberships);
```

Comment:
Per commentron repo, `channel_url` (aka `authorUri`) is an optional field, so it can be `undefined`.
When that happens, you are passing an array of undefined as the entry.